install.sh

```diff
@@ -4,33 +4,50 @@
 # Remove ads system-wide in Linux                         #
 ###########################################################
 # authors:      sedrubal, diy-electronics                 #
-# version:      v3.1                                      #
+# version:      v3.2                                      #
```

Comment:
Incrementing the version number is something the project maintainer should do. 😉